flake.lock

```diff
@@ -2,15 +2,18 @@
   "nodes": {
     "nixpkgs": {
       "locked": {
-        "lastModified": 1618628710,
-        "narHash": "sha256-9xIoU+BrCpjs5nfWcd/GlU7XCVdnNKJPffoNTxgGfhs=",
-        "path": "/nix/store/z1rf17q0fxj935cmplzys4gg6nxj1as0-source",
-        "rev": "7919518f0235106d050c77837df5e338fb94de5d",
-        "type": "path"
+        "lastModified": 1638371214,
+        "narHash": "sha256-0kE6KhgH7n0vyuX4aUoGsGIQOqjIx2fJavpCWtn73rc=",
+        "owner": "nixos",
+        "repo": "nixpkgs",
+        "rev": "a640d8394f34714578f3e6335fc767d0755d78f9",
+        "type": "github"
       },
       "original": {
-        "id": "nixpkgs",
-        "type": "indirect"
+        "owner": "nixos",
```

Comment:
Following `nixos-21.11` should be okay, right?

Reply:
Why do you need to do this?

Reply:
I updated to `21.11` because the previous revision doesn't seem to have `aarch64-darwin` support at all.

See [flake.nix of the previous revision](https://github.com/NixOS/nixpkgs/blob/7919518f0235106d050c77837df5e338fb94de5d/flake.nix#L14-L21)

Compare with [flake.nix of the new revision](https://github.com/NixOS/nixpkgs/blob/e34c5379866833f41e2a36f309912fa675d687c7/flake.nix#L14) where `lib.systems.supported.hydra` is defined [here](https://github.com/NixOS/nixpkgs/blob/e34c5379866833f41e2a36f309912fa675d687c7/lib/systems/supported.nix#L7-L24) and includes `aarch64-darwin`.